exporter/azure_monitor/azure_monitor.go

```diff
@@ -0,0 +1,46 @@
+package azure_monitor

```

Comment:
Every package should have a // Package comment to outline what the package is responsible for. If the package functionality is simple, a short comment can be made.

Reply:
Are you asking for something like this?
```suggestion
package azure_monitor
// Package: extension for exporters to Azure Monitor.
// This includes examples on how to create azure exporters to send spans.
```